<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputWidget</class>
 <widget class="QWidget" name="InputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CTB - Click The Button</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resouces.qrc">
    <normaloff>:/logo/logo/logo.ico</normaloff>:/logo/logo/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color:#f8f8f8;
background-color:#181818;</string>
  </property>
  <widget class="QFrame" name="frame_input_main">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>780</width>
     <height>70</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox_input_time">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="title">
     <string>Время</string>
    </property>
    <widget class="QLabel" name="label_input_time_remains">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>Осталось:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>секунд</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_input_speed">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>15</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>15</height>
     </size>
    </property>
    <property name="title">
     <string>Скорость</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>Текущая:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_input_speed_current_per_min">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>клик/мин</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_input_time_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="title">
     <string>Нажатия</string>
    </property>
    <widget class="QLabel" name="label_input_amount_clicks">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>50</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string>Кликов</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_input_save">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#383838;</string>
   </property>
   <property name="text">
    <string>Сохранить результаты</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_input_quit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>350</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#383838;</string>
   </property>
   <property name="text">
    <string>Выйти в главное меню</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_input_start">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>350</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#383838;</string>
   </property>
   <property name="text">
    <string>Начать игру</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_input_gamespace" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>780</width>
     <height>225</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #8f8f91;
</string>
   </property>
   <widget class="QPushButton" name="pushButton_active">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>80</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f6f6f6;
color: black;
border-radius: 25px;
border: 2px solid #8f8f91;
</string>
    </property>
    <property name="text">
     <string>PUSH
ME</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resouces.qrc"/>
 </resources>
 <connections/>
</ui>
